apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -146,19 +153,34 @@ export default class AnimationPickerBody extends React.Component {
     if (this.props.hideBackgrounds) {
       categories = categories.filter(category => category !== 'backgrounds');
     }
+    // Level-specific animations currently have the following categories:
```

Comment:
This animation is used in > 100 .level files with the category "animals" so the plan is to create a copy of it so it will also be displayed in the 'Costumes' category.